source/analysis/REIXS/REIXSXESImageInterpolationAB.h

```diff
@@ -61,6 +61,8 @@ class REIXSXESImageInterpolationAB : public AMStandardAnalysisBlock
 	Q_PROPERTY(int correlation2SmoothingType READ correlation2SmoothingType WRITE setCorrelation2SmoothingType)
 	Q_PROPERTY(int correlation2SmoothingMode READ correlation2SmoothingMode WRITE setCorrelation2SmoothingMode)
 
+	Q_PROPERTY(int binningLevel READ binningLevel WRITE setBinningLevel)

```

Comment:
If this is going to be persistently stored in the database, I would suggest making a default value (probably 1?).  

Use: `Q_CLASSINFO("binningLevel", "upgradeDefault=1")`